gcp/website/frontend_handlers.py

```diff
@@ -571,29 +574,36 @@ def get_vuln_count_for_ecosystem(ecosystem: str) -> int:
   return ecosystem_counts.get(ecosystem, 0)
 
 
-def osv_get_by_id(vuln_id):
+def osv_get_by_id(vuln_id) -> tuple[osv.Bug | None, str | None]:
   """Gets bug details from its id. If invalid, aborts the request."""
   if not vuln_id:
     abort(400)
-    return None
 
   bug = osv.Bug.get_by_id(vuln_id)
   if not bug:
-    if osv.ImportFinding.get_by_id(vuln_id):
-      # abort(404) is too simplistic for this.
-      raise VulnerabilityNotImported(vuln_id)
-    # abort(404) is too simplistic for this.
-    raise VulnerabilityNotFound(vuln_id)
+    alias_group = osv.AliasGroup.query(osv.AliasGroup.bug_ids == vuln_id).get()
```

Comment:
might be worth putting this in its own function to make this function easier to understand.